app/models/appointment_slot_preset.rb

```diff
@@ -1,8 +1,29 @@
 class AppointmentSlotPreset < ActiveRecord::Base
+  include PushUpdatesMinimal
+
+  validate :no_conflict, on: :update
```

Comment:
Not on `create` as well?

Reply:
on create the slots actually get merged together, the edge case was on update only